js/index.js

```diff
@@ -0,0 +1,202 @@
+const fridgeIcon = L.icon({
+  iconUrl: 'data/fridgeicon.png',
+  iconSize: [20, 20],
+  iconAnchor: [15, 15],
+  popupAnchor: [0, -15],
+});
+
+const shelterIcon = L.icon({
+  iconUrl: 'data/sheltericon.png',
+  iconSize: [20, 20],
+  iconAnchor: [15, 15],
+  popupAnchor: [0, -15],
+});
+
+const layerOptions = {
+  '1-PA-slide': {
+    style: (feature) => {
+      const getColor = (FI_Rate) => {
+        return FI_Rate > 0.144 ? '#ff0a0a' :
+           FI_Rate > 0.131 ? '#ff6600' :
+           FI_Rate > 0.123 ? '#ff9b00' :
+           FI_Rate > 0.114 ? '#ffcb00' :
+           '#fff700';
+      };
+      const borderColor = feature.properties.County === 'Philadelphia County, Pennsylvania' ? 'blue' : 'black';
+      const borderWeight = feature.properties.County === 'Philadelphia County, Pennsylvania' ? 3 : 2;
+      return {
+        fillColor: getColor(feature.properties.FI_Rate),
+        weight: borderWeight,
+        opacity: 1,
+        color: borderColor,
+        fillOpacity: 0.7,
+      };
+    },
+  },
+  '2-Poverty-slide': {
+    style: (feature) => {
+      const getColor = (PercPoor) => {
+        return PercPoor > 0.33816 ? '#1527c8' :
+        PercPoor > 0.23302 ? '#425dcf' :
+        PercPoor > 0.15836 ? '##6f93d7' :
+        PercPoor > 0.09148 ? '#9bc9de':
+             '#c8ffe5';
+      };
+      return {
+        fillColor: getColor(feature.properties.PercPoor),
+        weight: 2,
+        opacity: 1,
+        color: 'black',
+        fillOpacity: 0.7,
+      };
+    },
+    onEachFeature: (feature, layer) => {
+      // Calculate the poverty percentage and round it to two decimal places
+      const povertyPercentage = (feature.properties.PercPoor * 100).toFixed(2);
+      // Bind the tooltip to the polygon
+      layer.bindTooltip(`Poverty: ${povertyPercentage} %`, {
+        permanent: false, // Tooltip appears on hover
+        direction: 'top', // Tooltip position
+      });
+    },
+  },
+  '3-CB-slide': {
+    style: (feature) => {
+      const getColor = (PercCB) => {
+        return PercCB > 0.62828 ? '#1527c8' :
+        PercCB > 0.53602 ? '#425dcf' :
+        PercCB > 0.44682 ? '##6f93d7' :
+        PercCB > 0.34192 ? '#9bc9de':
+             '#c8ffe5';
+      };
+      return {
+        fillColor: getColor(feature.properties.PercCB),
+        weight: 2,
+        opacity: 1,
+        color: 'black',
+        fillOpacity: 0.7,
+      };
+    },
+    onEachFeature: (feature, layer) => {
+      const costburdenPercentage = (feature.properties.PercCB * 100).toFixed(2);
+      layer.bindTooltip(`Cost-Burdened: ${costburdenPercentage} %`, {
+        permanent: false,
+        direction: 'top',
+      });
+    },
+  },
+  '4-Access-slide': {
+    style: (feature) => {
+      const getColor = (TOTAL_HPSS) => {
+        return TOTAL_HPSS > 5 ? '#3b0d52' :
+        TOTAL_HPSS > 3.25 ? '#6a3979' :
+        TOTAL_HPSS > 2.25 ? '#9a66a3' :
+        TOTAL_HPSS > 1 ? '#cc96cf':
+             '#ffc9fd';
+      };
```

Comment:
**question:** I found the switch in color palette and style for this slide a little jarring. Is there a reason that you went with the switch here?